webapp/tests/forms/steps/test_steuerlotse_step.py

```diff
@@ -0,0 +1,469 @@
+            steuerlotse_step.name = "This is the one"
+            steuerlotse_step._pre_handle(None)
+
+            steuerlotse_step.render_info.overview_url = overview_url
+
+            expected_render_info = RenderInfo(step_title=title, step_intro=intro, form=None, prev_url=None,
+                                              next_url=None, submit_url=url_for(endpoint=correct_endpoint,
+                                                                                step=steuerlotse_step.name,
+                                                                                link_overview=steuerlotse_step.has_link_overview),
+                                              overview_url=overview_url)
+
+            self.assertEqual(expected_render_info, steuerlotse_step.render_info)
+
+    def test_pre_handle_returns_stored_data_untouched(self):
+        data = {'father': 'Mufasa'}
+
+        with app.app_context() and app.test_request_context():
+            steuerlotse_step = SteuerlotseStep(title="The Steuerlotse I", intro="Not so long ago....", endpoint="lotse")
+            steuerlotse_step.name = "This is the one"
+
+            return_stored_data = steuerlotse_step._pre_handle(data)
+
+            self.assertEqual(data, return_stored_data)
+
+
+class TestSteuerlotseStepPostHandle(unittest.TestCase):
+
+    def test_if_post_handle_called_then_return_render_result(self):
+        stored_data = {'location': "Spiro's tower", 'attacker': 'Mulch Diggums', 'target': 'C Cube'}
+        with app.app_context() and app.test_request_context():
+            steuerlotse_step = SteuerlotseStep(title="The Steuerlotse I", intro="Not so long ago....", endpoint="lotse")
+            steuerlotse_step.name = "This is the one"
+            steuerlotse_step._pre_handle(stored_data)
+
+            with patch("app.forms.steps.steuerlotse_step.SteuerlotseStep.render", MagicMock()) as step_render:
+                post_result = steuerlotse_step._post_handle(stored_data)
+
+                self.assertEqual(step_render.return_value, post_result)
+                step_render.assert_called()
+
+    def test_if_redirection_url_set_then_return_redirect(self):
+        stored_data = {'location': "Spiro's tower", 'attacker': 'Mulch Diggums', 'target': 'C Cube'}
+        with app.app_context() and app.test_request_context():
+            steuerlotse_step = SteuerlotseStep(title="The Steuerlotse I", intro="Not so long ago....", endpoint="lotse")
+            steuerlotse_step.name = "This is the one"
+            steuerlotse_step._pre_handle(stored_data)
+
+            redirect_url = url_for(endpoint="lotse", step="RedirectToThis",
+                                   link_overview=steuerlotse_step.has_link_overview)
+            steuerlotse_step.render_info.redirect_url = redirect_url
+
+            with patch("app.forms.steps.steuerlotse_step.SteuerlotseStep.render", MagicMock()) as step_render:
```

Comment:
We need to patch this because the default step does nothing, right? I have two suggestions: a) Replace this with a Mock Step deriving from SteuerlotseStep that has a render method or b) Add a comment to explain this (and in both cases remove the  `as step_render`) - I think I'd actually prefer approach b).

Reply:
I agree with you and would also vote for approach b